lua/nvim-drawer/init.lua

```diff
@@ -774,7 +810,10 @@ function mod.find_instance_for_winid(winid)
   end
 end
 
-function mod.setup(_)
+--- @param options? NvimDrawerSetupOptions
+function mod.setup(options)
+  current_options = vim.tbl_deep_extend('force', default_options, options or {})
```

Comment:
Ideally `side_order` would be sanitized here, so if people don't specify the whole list, the missing entries are inserted at the beginning.